src/Php55/README.md

```diff
@@ -0,0 +1,17 @@
+Symfony Polyfill / Php55
+========================
+
+This component provides functions unavailable in releases prior to PHP 5.5:
+
+- `boolval`
+- `json_last_error_msg`
+- `array_column`
+- `hash_pbkdf2`
```

Comment:
the password_* functions are also provided (because ircmaxell/passsword_compat is required)